src/approaches/open_loop_llm_approach.py

```diff
@@ -1,14 +1,13 @@
-"""Open-loop large language model (LLM) meta-controller approach.
+"""Open-loop large language model (LLM) meta-controller approach. Example
+command line: export OPENAI_API_KEY=<your API key> python src/main.py.
```

Comment:
can we move the `python src/main.py` to a newline so that it's obvious that the `export` command needs to be run separately?